src/config/libinput.c

```diff
@@ -22,6 +22,7 @@ libinput_category_init(struct libinput_category *l)
 	l->accel_profile = -1;
 	l->middle_emu = -1;
 	l->dwt = -1;
+	l->click_method = LIBINPUT_CONFIG_CLICK_METHOD_BUTTON_AREAS;
```

Comment:
We should set this to `-1` by default so that we don't configure devices unless the requested.

Reply:
This also implies not using `enum libinput_config_click_method` as data type.

Reply:
done.